src/lib/util.js

```diff
@@ -37,9 +37,9 @@ function absoluteUrlValidator(url) {
 
     const [owner, repo] = relativeRepoUrl.split("/");
 
-    if (!url.toLowerCase().includes("github.com"))
+    if (urlObject.hostname.toLowerCase() !== "github.com") {
       return [false, null];
```

Comment:
This looks almost right, I would just remove the `.toLowerCase()` method, because the URL constructor does it for us.

Reply:
Could have sworn it failed the test suite. I'll change it and make sure.